### PR TITLE
Initialize project w components

### PR DESCRIPTION
**Pull request by AlexHamn**
### Changes in this branch

- Initialise React app.
- Add React Redux (`npm install react-redux`).
- Structure application files using a "feature folder" approach and use the [*ducks* pattern](https://github.com/erikras/ducks-modular-redux) for Redux files. Files/folder structure look similar to this:
    ```    
    ...
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js
    ``` 
- The *building blocks* of the app are set as re-usable components.
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two `<Route>`s and `<Link>`s for the app's navigation:
    - Books - the default view
        - Display the list of books (empty at this point but it is ready for the data) with the Remove button (no functionality yet)
        - Have a form for adding a book (no functionality yet)
    - Categories
        - Display "Under construction" text only.
- Styling is not required at this point.

### CODE REVIEW

**Comment by AlexHamn** [comment]
I am aware that my workflow is not working. Would you mind taking a look at it and helping me fix it? :pray:  :pray: 
I am not managing to figure it out :sweat:

**Comment by mke2111** [comment]
For this error, it is usually due to the `package-lock.json` file. The solution that you might need is to delete this file, commit and push your changes and it should be cleared. Give it a try. Always check the [QUESTIONS](https://questions.microverse.org/c/react/9) on discourse, a solution usually exists.

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @AlexHamn, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <form>
+      <br />
+      <input type="text" id="addBook" name="addBook" placeholder="new book" />
+      <select name="category" id="category">
+        <option value="Drama" selected>Drama</option>
+        <option value="Economy" selected>Economy</option>
+        <option value="Science Fiction">Science Fiction</option>
+        <option value="Action">Action</option>
+      </select>
+      <input type="submit" value="Submit" />
```
Comment (mke2111) [inline src/components/AddBook.js]: - [x] This component should render a form with inputs for both the title and author of the book. You're currently having only the title input. Add the author input to your form.

**Review by mke2111 — APPROVED** [review]
### Hi @AlexHamn, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**